vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -227,7 +227,7 @@ func (cluster *ClusterState) AddOrUpdateVpa(apiObject *vpa_types.VerticalPodAuto
 	}
 	vpa.Conditions = conditionsMap
 	vpa.Recommendation = currentRecommendation
-	vpa.ResourcePolicy = apiObject.Spec.ResourcePolicy
+	vpa.ScalingPolicy = apiObject.DeepCopy()
```

Comment:
I wonder - the ScalingPolicy contains Recommendation too (via GetRecommendation() method); what is the relation between this one & vpa.Recommendation?